hugolib/site.go

```diff
@@ -956,14 +956,14 @@ func (s *Site) handleDataFile(r source.ReadableFile) error {
 			higherPrecedentMap := higherPrecedentData.(map[string]interface{})
 			for key, value := range data.(map[string]interface{}) {
 				if _, exists := higherPrecedentMap[key]; exists {
-					s.Log.WARN.Printf("Data for key '%s' in path '%s' is overridden higher precedence data already in the data tree", key, r.Path())
+					s.Log.INFO.Printf("Data for key '%s' in path '%s' is overridden by higher precedence data already in the data tree", key, r.Path())
```

Comment:
I have never seen these warnings (I have changed to `stdoutThreshold = jww.LevelWarn` in my local builds for quite some time now). From the look of those messages, they deserve to be warnings.

Reply:
.. and same for the other two "overridden by .." messages in this file.

Reply:
Maybe.

Reply:
The thing is, is you have:

```
/data/foo.json with mykey
(and)
/theme/data/foo.json with mykey
```

The above will then trigger a WARN, which I would find very annoying and definitively not what I wanted. This is not my code, but I think the only way you can get this within the project /data is  like this:

/data/foo.json 
/data/foo.toml

But then I think you have shot yourself in the foot and deserves it.

Reply:
Actually no.  See #4379.  I spent a lot of time writing the analysis and the test code in that pull request demonstrating everything.  Oh well. :no_mouth:

Reply:
Actually no to what?

Reply:
Note that the "data dif functionality" has been rewritten since #4379.

Reply:
OK, I have read the issue. The warnings above can happen in both valid and potentially invalid scenarios. Which is a pickle.

Reply:
I'll bump it to WARN for now, and handle the "overlap issue" in another issue.